.github/workflows/ci.yml

```diff
@@ -6,6 +6,9 @@ on:
       - main
   pull_request:
 
+env:
+  GITHUB_TOKEN: ${{ secrets.OPENAPI_TS_BOT_GITHUB_TOKEN }}
```

Comment:
Note: this was added because Turborepo adds a “shield” around env vars and doesn’t accidentally expose something sensitive, so you declare what to expose. But the flipside is it throws an error when this _isn’t_ set. I may be missing a configuration setting, but until then, this was simpler.